docs/wiki/deployment/process-auditing.md

```diff
@@ -8,28 +12,93 @@ to your query schedule, or to a query pack.
+The only prerequisite for using osquery's auditing functionality on Linux is that you must use a kernel version that contains the audit functionality. Most kernels over version 2.6 have this capability.
 
-How does this work? Let's walk through 3 configuration options. These can be set at the [command line](../installation/cli-flags.md), or via the configuration's options. 
+There is no requirement to install auditd or libaudit. Osquery only uses the audit features that exist in the kernel.
+
+A sample log entry from process_events may look something like this:
+```
+{
+  "action": "added",
+  "columns": {
+    "uid": "0",
+    "time": "1527895541",
+    "pid": "30219",
+    "path": "/usr/bin/curl",
+    "auid": "1000",
+    "cmdline": "curl google.com",
+    "ctime": "1503452096",
+    "cwd": "",
+    "egid": "0",
+    "euid": "0",
+    "gid": "0",
+    "parent": ""
+  },
+  "unixTime": 1527895550,
+  "hostIdentifier": "vagrant",
+  "name": "process_events"
+}
+```
 
-1. `--disable_audit=false` by default this is set to `true` and prevents osquery from opening the kernel audit's netlink socket. 
-2. `--audit_allow_config=true` by default this is set to `false` and prevents osquery from making audit configuration changes. These changes include adding/removing rules, setting the global enable flags, and adjusting performance and rate parameters.
-3. `--audit_persist=true` but default this is `true` and instructs osquery to 'regain' the audit netlink socket if another process also accesses it.
+To better understand how this works, let's walk through 4 configuration options. These flags can be set at the [command line](../installation/cli-flags.md) or placed into the osquery.flags file.
 
-On Linux a companion table `user_events` is included that provides several authentication-based events. If you are enabling process auditing it should be trivial to also include this table.
+1. `--disable_audit=false` by default this is set to `true` and prevents osquery from opening the kernel audit's netlink socket. By setting it to `false`, we are telling osquery that we want to enable auditing functionality.
+2. `--audit_allow_config=true` by default this is set to `false` and prevents osquery from making changes to the audit configuration settings. These changes include adding/removing rules, setting the global enable flags, and adjusting performance and rate parameters. Unless you plan to set all of those things manually, you should leave this as true. If you are configuring audit, using a control binary, or `/etc/audit.conf`, your osquery *may* override your settings.
+3. `--audit_persist=true` but default this is `true` and instructs osquery to 'regain' the audit netlink socket if another process also accesses it. However, you should do your best to ensure there will be no other program running which is attempting to access the audit netlink socket.
+4. `--audit_allow_process_events=true` this flag indicates that you would like to record process events
```

Comment:
I would also suggest keeping this line `-If you would like to debug the audit logging use the hidden flag `--audit_debug`. This will print all of the RAW audit lines to osquery's stdout.` as a residual note, just to give people insight that there's debug flags.

Reply:
Nvm, I see you have that below :)